chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -91,5 +92,5 @@ def step(context, message):
         ok_(message not in "".join(context.stdout.readlines()))
     except AssertionError:
         context.stdout.seek(0)
-        print context.stdout.readlines()
+        print(context.stdout.readlines())
```

Comment:
was necessary because the relevant tests were failing without it for some obscure reason